### PR TITLE
adding scal monitoring to online dqm

### PR DESCRIPTION
**Pull request by sachikot**
adding scal monitoring to online dqm for 76X

### CODE REVIEW

File: DQMServices/Components/src/DQMScalInfo.cc [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+}
+
+DQMScalInfo::~DQMScalInfo(){
+}
+
+void DQMScalInfo::bookHistograms(DQMStore::IBooker & ibooker,
+                                edm::Run const & /* iRun */,
+                                edm::EventSetup const & /* iSetup */) {
+  std::cout << "histo booking" << std::endl;
```
Comment (deguio) [inline DQMServices/Components/src/DQMScalInfo.cc]: cout is not allowed. please remove

File: DQMServices/Components/src/DQMScalInfo.cc [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+}
+
+void DQMScalInfo::analyze(const edm::Event& e, const edm::EventSetup& c){
+  makeL1Scalars(e);
+  return;
+}
+
+void
+DQMScalInfo::endLuminosityBlock(const edm::LuminosityBlock& l, const edm::EventSetup& c)
```
Comment (deguio) [inline DQMServices/Components/src/DQMScalInfo.cc]: not used, please remove

**Comment by deguio** [comment]
thanks @sachikot 
please address my comments for all the branches. then I would say it can go it.
cheers,
F.

**Comment by deguio** [comment]
-1
rejecting for now. more commits needed.

**Comment by danduggan** [comment]
@cmsbuild please test
@sachikot, thanks for updating the code

**Comment by deguio** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**